lib/chef/resource/archive_file.rb

```diff
@@ -72,11 +85,11 @@ class ArchiveFile < Chef::Resource
       alias_method :extract_options, :options
       alias_method :extract_to, :destination
 
-      require "fileutils" unless defined?(FileUtils)
-
       action :extract do
         description "Extract and archive file."
 
+        require_libarchive
```

Comment:
Any particular reason this needs to be a method instead of a plain `require` here?

Reply:
The specs were exploding when I just did a require here.

Reply:
Weird